templates/form.html

```diff
@@ -86,7 +86,7 @@ <h1 class = "title" >Username Availability Checker</h1>
         </div>
     </div>
     <div class="footer">
-        Made with love in New Delhi.
+        <p>Made by <span><a href="https://github.com/manu-chroma/username-availability-checker/graphs/contributors"> Google Code-In mentors and students</a></span> of coala, from all around the world.</p>
```

Comment:
Hey,
There seems to be a redundant space here.
You may remove the space after the `href` attribute; before `Google Code-In`.

Reply:
```suggestion
        <p>Made by <span><a href="https://github.com/manu-chroma/username-availability-checker/graphs/contributors"> Google Code-In mentors and students</a></span> of coala and some other good friends, from all around the world.</p>
```

There have been some contributors too. I think this one's a little better 😉